### PR TITLE
feat: more accurate fst names

### LINKED ISSUE #17: Feature request: create dictionary descriptive analyzer that lacks +Err/Frag analyses
Produce a descriptive analyzer that eliminates `+Err/Frag` analyses, as well as removes the `+Err/Orth` tag.
This FST could be created in `quick.mk`; not necessarily required in the primary build system.

This will elimate downstream applications (namely, _itwêwina_) from having to manually filter out `+Err/Frag` analyses.

@aarppe: any comments?

### LINKED ISSUE #18: More accurate labeling of FSTs?
@eddieantonio I've been pondering about this for a while, namely that the terms `descriptive` and `normative` are not really accurately descriptors of the varying FSTs. Rather, the terms `strict` (or `standard`) and `relaxed` might better represent the relationship of the FSTs with normative/standard orthography. Perhaps we could follow this with the FSTs created with `quick.mk`.

### PR DESCRIPTION
**Pull request by eddieantonio**
Work/life balance? What work/life balance! 😜

I have renamed the FSTs, and now here are the ones we'll use:

 - `crk-strict-analyzer`: analyzer, no spelling relaxation
 - `crk-strict-generator-with-morpheme-boundaries`: given an analyses, produces forms with `<` and `>` word boundaries!
 - `crk-strict-generator`: same as above, the morpheme boundaries are omitted
- `crk-relaxed-analyzer`: analyzer with spelling relaxation

And some new FSTs!

 - `crk-strict-analyzer-for-dictionary`: analyzer lacking `+Err/Frag` forms
 - `crk-relaxed-analyzer-for-dictionary`: analyzer lacking `+Err/Frag` forms and omitting the `+Err/Orth` tag from analyses

This is the series of commands issued when you run `make -f quick.mk dicts`:

```sh
echo "Concatenating LEXC files."
cat fst/root.lexc fst/affixes/noun_affixes.lexc fst/affixes/verb_affixes.lexc fst/stems/noun_stems.lexc fst/stems/numerals.lexc fst/stems/particles.lexc fst/stems/pronouns.lexc fst/stems/verb_stems.lexc fst/stems/derivation_stems.lexc fst/stems/non_standard.lexc fst/stems/noun_vocatives.lexc > crk-dict.lexc
echo "Compiling LEXC code."
hfst-lexc --format=openfst-tropical -o crk-lexc-dict.hfst crk-dict.lexc
echo "Compiling xfscript code for morphophonology."
echo "(this one takes a while... ☕️)"
hfst-xfst --pipe-mode --verbose --format=openfst-tropical	-e "source fst/phonology.xfscript" -E "save stack crk-phon.hfst"
echo "Composing lexicon with morphophonology."
hfst-compose --harmonize-flags -1 crk-lexc-dict.hfst -2 crk-phon.hfst | hfst-minimize - -o crk-strict-generator-with-morpheme-boundaries.hfst
echo "Compiling filter to remove morpheme boundaries."
echo '%> -> 0, %< -> 0;' | hfst-regexp2fst --verbose --semicolon -o remove-morpheme-boundary-filter.hfst
echo "Removing morpheme boundaries to create strict analyzer."
hfst-compose --harmonize-flags -1 crk-strict-generator-with-morpheme-boundaries.hfst -2 remove-morpheme-boundary-filter.hfst | hfst-invert | hfst-minimize -o crk-strict-analyzer.hfst
hfst-fst2fst --optimized-lookup-unweighted -i crk-strict-analyzer.hfst -o crk-strict-analyzer.hfstol
echo "Compiling regular expression implementing spelling-relaxation."
hfst-regexp2fst --verbose --semicolon orthography/spellrelax.regex -o crk-orth.hfst
echo "Composing spelling relaxation transducer with normative analyzer transducer to create descriptive analyzer."
hfst-invert -i crk-strict-analyzer.hfst | hfst-compose --harmonize-flags -1 - -2 crk-orth.hfst | hfst-minimize | hfst-invert - -o crk-relaxed-analyzer.hfst
hfst-fst2fst --optimized-lookup-unweighted -i crk-relaxed-analyzer.hfst -o crk-relaxed-analyzer.hfstol
echo "Inverting the analyzer to create the strict generator."
hfst-invert crk-strict-analyzer.hfst -o crk-strict-generator.hfst
hfst-fst2fst --optimized-lookup-unweighted -i crk-strict-generator.hfst -o crk-strict-generator.hfstol
hfst-fst2fst --optimized-lookup-unweighted -i crk-strict-generator-with-morpheme-boundaries.hfst -o crk-strict-generator-with-morpheme-boundaries.hfstol
echo "Compiling filter to remove +Err/Frag analyses and omit +Err/Orth from analyses"
echo '~[ $[ "+Err/Frag" ] ] .o. [ "+Err/Orth" -> 0 ] ; ' | hfst-regexp2fst --verbose --semicolon - -o crk-dict-filter.hfst
echo "Creating dictionary version of crk-strict-analyzer.hfst"
hfst-compose  --harmonize-flags -1 crk-strict-analyzer.hfst -2 crk-dict-filter.hfst |\
		hfst-minimize - -o crk-strict-analyzer-for-dictionary.hfst
hfst-fst2fst --optimized-lookup-unweighted -i crk-strict-analyzer-for-dictionary.hfst -o crk-strict-analyzer-for-dictionary.hfstol
echo "Creating dictionary version of crk-relaxed-analyzer.hfst"
hfst-compose  --harmonize-flags -1 crk-relaxed-analyzer.hfst -2 crk-dict-filter.hfst |\
		hfst-minimize - -o crk-relaxed-analyzer-for-dictionary.hfst
hfst-fst2fst --optimized-lookup-unweighted -i crk-relaxed-analyzer-for-dictionary.hfst -o crk-relaxed-analyzer-for-dictionary.hfstol
```

Fixes #17 
Fixes #18

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,16 +1,111 @@
...
+
+### morpheme boundary
+
+A mark that denotes where one meaningful sub-unit of a word is separated
+from a different, adjacent part of the same word.
+
+In Plains Cree, these marks are `<` and `>`. `<` occurs before the stem
+of a verb, separating person marking to the left of it, and `>` is
+placed immediately after the verb stem, separating the _other_ person
+marking.
```
Comment (aarppe) [inline src/README.md:75]: The morpheme boundary does apply to noun affixation and stems as well.

File: src/README.md [hunk 2]
```diff
@@ -1,16 +1,111 @@
...
+dictionary for Plains Cree.
+
+In practice, this means analyzers lack all `+Err/Frag` analyses and omit
+the `+Err/Orth` tag, since these offer little value to the dictionary
+(it's trying to match a lemma, after all!).
+
+### morpheme boundary
+
+A mark that denotes where one meaningful sub-unit of a word is separated
+from a different, adjacent part of the same word.
```
Comment (aarppe) [inline src/README.md:70]: image

File: src/README.md [hunk 3]
```diff
@@ -1,16 +1,111 @@
...
+ - ki<wâpam>âw
+
+
+### strict
+
+Only produce or recognize text in a strict orthographical norm or
+standard. In the case of Plains Cree, this means texts that adheres to
+the [standard Roman orthography (SRO)][SRO].
+
+Note: The itwêwina database of dictionary heads is internally stored in SRO.
```
Comment (aarppe) [inline src/README.md:92]: using circumfix as the diacritic marker of vowel lenght.

File: src/README.md [hunk 4]
```diff
@@ -1,16 +1,111 @@
...
+| ewapamat   | ê-wâpamat _or_ ê-wâpamât | ᐁ ᐚᐸᒪᐟ      |
+
+[SRO]: https://creeliteracy.org/wp-content/uploads/2016/01/htsiic-covers-nocontacts.pdf
+
+### relaxed
+
+(of an **analyzer**) it can take in forms that do not conform to the
+orthographical standard. We apply _spelling relaxation_ such that we can
+accept multiple possible spellings of words, and not require one exact
+spelling of words. The spelling relaxation is done through a manually
```
Comment (aarppe) [inline src/README.md:110]: accept multiple possible **variant** spellings of words, and will not require **adhering exactly to one normative
spelling of a** word.

**Comment by eddieantonio** [comment]
@aarppe changes done! Are the new FST names okay?

**Comment by aarppe** [comment]
It was my suggestion, and the pairing `strict` vs. `relaxed`, in relation to the orthographical standard/norm, is one that has emerged from my thinking over a year or two.

Since we'd want to stick with these once we make the change, perhaps we could do a sanity check in Slack, just to check that we haven't missed out on any wrong association.

**Comment by aarppe** [comment]
Ok - based on the lack of expressed concerns in Slack, we can move forward with this change.

**Comment by aarppe** [comment]
Could I have simply merged this branch? There was no explicit way to approve the change.

**Comment by eddieantonio** [comment]
Yes, you could have pressed the big ol' green merge button!

However, there is a way of approving the changes. Click on the "Files changed" tab, then there's a "Review changes" button and there's an "🔘  Approve" button. The UI is a little bit hidden, for sure. See here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews

**Comment by aarppe** [comment]
I thought so - I did recall seeing Review/Approve buttons somewhere, but wasn't able to locate those easily.

**Pull request merged**